.gitignore

```diff
@@ -81,3 +81,10 @@ graph.png
 
 # Mac OSX
 .DS_Store
+
+# Eclipse Stuff
+.project
+.pydevproject
+
+# Distribution
+/distribution/
```

Comment:
Where is distribution folder coming from? if it is romcon/distribution it shouldn't be in the same directory as PFNET